### PR TITLE
wil: correct package_info and msvc version

### PR DESCRIPTION
**Pull request by prince-chrismc**
review mistakes from https://github.com/conan-io/conan-center-index/pull/16701#discussion_r1166056823

Specify library name and version:  **lib/1.0**

---

- [ ] I've read the [contributing guidelines](https://github.com/conan-io/conan-center-index/blob/master/CONTRIBUTING.md).
- [ ] I've used a [recent](https://github.com/conan-io/conan/releases/latest) Conan client version close to the [currently deployed](https://github.com/conan-io/conan-center-index/blob/master/.c3i/config_v1.yml#L6).
- [ ] I've tried at least one configuration locally with the [conan-center hook](https://github.com/conan-io/hooks.git) activated.

### CODE REVIEW

**Comment by uilianries** [comment]
@prince-chrismc please, avoid creating custom branches from CCI repo directly and prefer your fork instead, it starts new CI jobs for each new branch 😄

**Review by uilianries — APPROVED** [review]
LGTM

**Comment by prince-chrismc** [comment]
Whoops! sorry, that was an accident 👾 good catch did not even see it

**Review by SSE4 — APPROVED** [review]

File: recipes/wil/all/conanfile.py [hunk 1]
```diff
@@ -33,21 +33,20 @@ def _min_cppstd(self):
...
     def export_sources(self):
         export_conandata_patches(self)
 
     def layout(self):
         basic_layout(self, src_folder="src")
 
-    # same package ID for any package
     def package_id(self):
-        self.info.clear()
+        self.info.clear() # same package ID for any package
```
Comment (SpaceIm) [inline recipes/wil/all/conanfile.py:49]: this comment is a truism.

**Pull request merged**